doc/fping.pod

```diff
@@ -238,6 +240,8 @@ Ignored (for compatibility with fping 2.4).
 
 Show targets that are unreachable.
 
+Not compatible with option B<--count> / B<-c>.
```

Comment:
How about adding something like "(Option B<-c> or B<-C> overrides B<-u>)"? This could start in the same line as the existing description.